go.mod

```diff
@@ -0,0 +1,55 @@
+module github.com/maiqueb/persistentips
```

Comment:
my plan is to keep this in a personal repo, then "donate" the CRD to the k8snetworkplumbingwg.

Plus push an update proposal to the kubernetes multi network de-facto standard.